### DISCUSSION TITLE
Daikin Air to water heat pump x10a connectors differ from i've seen on here.

### DISCUSSION THREAD

**Discussion by vindiv8** [raomin/ESPAltherma#185] [General]
My Daikin board looks different to what  I've seen on here, the x10a seems to be populated and a different connector.  The x11a and x12a look more like what people are using.

Should I be using the x12a in my case?
![PXL_20221105_152211081](https://user-images.githubusercontent.com/117530119/200548532-701b2215-682e-46fd-85af-e929e2aa9a0a.jpg)

Thank you
Kev

**Comment by alexdelli** [comment]
> My Daikin board looks different to what I've seen on here, the x10a seems to be populated and a different connector. The x11a and x12a look more like what people are using.
> 
> Should I be using the x12a in my case? ![PXL_20221105_152211081](https://user-images.githubusercontent.com/117530119/200548532-701b2215-682e-46fd-85af-e929e2aa9a0a.jpg)
> 
> Thank you Kev

Yes, same wiring

**Reply by vindiv8** [reply]
So connect up to the x12a, yes?
Thank you!

**Reply by alexdelli** [reply]
Yes, see here 
https://github.com/raomin/ESPAltherma/issues/40#issuecomment-1284360843

**Reply by vindiv8** [reply]
Awesome, yes that's the one. Thank you.
